certificateshop-logic/src/main/java/com/epam/esm/dao/ListToSetConverter.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.esm.dao;
+
+import java.sql.ResultSet;
+import java.util.List;
+
+public interface ListToSetConverter {
+    ResultSet getResultSet(List<String> headers, List<List<Object>> data) throws Exception;
+}
```

Comment:
This is so weird method name for ListToSetConverter because I expect to see some convert... method in the convertor rather than getResultSet

Reply:
That is the point: the method converts List<List<Object>> to ResultSet.

Reply:
Done